src/hotspot/share/gc/parallel/mutableNUMASpace.cpp

```diff
@@ -160,7 +160,7 @@ size_t MutableNUMASpace::tlab_capacity(Thread *thr) const {
     }
   }
   // That's the normal case, where we know the locality group of the thread.
-  int i = lgrp_spaces()->find(&lgrp_id, LGRPSpace::equals);
+  int i = lgrp_spaces()->find((uint*)&lgrp_id, LGRPSpace::equals);
```

Comment:
I guess it is somewhat outside the scope of this PR but I wish this code would make its mind up about whether the NUMA group ids are `int` or `uint`!